components/Converters/samples/StringVisibilityConverterSample.xaml

```diff
@@ -0,0 +1,21 @@
+﻿<!--  Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE file in the project root for more information.  -->
+<Page x:Class="ConvertersExperiment.Samples.StringVisibilityConverterSample"
+      xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+      xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+      xmlns:converters="using:CommunityToolkit.WinUI.Converters"
+      xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+      xmlns:local="using:ConvertersExperiment.Samples"
+      xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+      mc:Ignorable="d">
+
+    <Page.Resources>
+        <converters:StringVisibilityConverter x:Key="StringVisibilityConverter"
+                                              EmptyValue="Visible"
+                                              NotEmptyValue="Collapsed" />
```

Comment:
Are these defaults or inverted? If inverted, should we use ConverterParameter instead?

Since many of these converters are similarly built/have base classes, maybe we need a section at the top with some of these general implementation notes. e.g. either can change values for true/false empty/notempty etc... or can use converterparameter to negate result...